src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -77,14 +102,14 @@ class RatedDisabilityList extends React.Component {
     if (!this.props.ratedDisabilities) {
       return <LoadingIndicator message="Loading your information..." />;
     }
-    // Display error message based on error type.
     if (
       this.props.ratedDisabilities.errors ||
       this.props?.ratedDisabilities?.ratedDisabilities.length === 0
```

Comment:
And do we actually need this line? if the first check is false but this one is true then line 109 will set `code` to undefined and the call to `noDisabilityRatingContent()` will not work correctly. Can you take another look at the logic in this conditional to make sure it's solid?

Reply:
Eeek good catch. We do need _something_ to check the length of the array, since we noticed that there are instances when the frontend receives a 200 response, but evss sends along an empty array. Maybe specifying a default parameter for `noDisabilityRatingContent()` would be the way to go here.